app/code/Magento/Ui/Component/Filters/Type/Select.php

```diff
@@ -36,23 +41,36 @@ class Select extends AbstractFilter
     /**
      * @param ContextInterface $context
      * @param UiComponentFactory $uiComponentFactory
-     * @param \Magento\Framework\Api\FilterBuilder $filterBuilder
+     * @param FilterBuilder $filterBuilder
      * @param FilterModifier $filterModifier
+     * @param BookmarkManagementInterface $bookmarkManagement
+     * @param RequestInterface $request
      * @param OptionSourceInterface|null $optionsProvider
      * @param array $components
      * @param array $data
      */
     public function __construct(
         ContextInterface $context,
         UiComponentFactory $uiComponentFactory,
-        \Magento\Framework\Api\FilterBuilder $filterBuilder,
+        FilterBuilder $filterBuilder,
         FilterModifier $filterModifier,
+        BookmarkManagementInterface $bookmarkManagement,
```

Comment:
Please, inject new dependencies according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)